src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -4,87 +4,95 @@ short-title: Integrate Flutter
-   f Creates the Flutter project that is hosted in the existing app.
- 1. [Add Module Dependency in Existing
-    App](#make-the-host-app-depend-on-the-flutter-module)
-    Sets up Xcode to include your Flutter project into the build of your
-    existing app.
+From the command line, run:
 
-## Create a Flutter module
+```terminal
+cd some/path/
+flutter create --template module my_flutter
+```
 
-Flutter projects created using `flutter create xxx` include simple host apps for
-your Flutter/Dart code (a single-ViewController iOS host). You can modify these
-host apps to suit your needs and build from there.
+TODO(jmagman): IDE instructions
 
-If you're starting off with an *existing* host app for either platform, you'll
-likely want to include your Flutter project in that app as some form of library
-instead.
+This creates a Flutter module project at `some/path/my_flutter/`.
 
-In order to embed Flutter as a library, use the Flutter module template.
-Executing `flutter create -t module xxx` produces a Flutter project with a
-CocoaPods pod designed for consumption by your existing host app.
+In that directory you can run the same `flutter` commands you would
+in any other Flutter project, like `flutter run --debug` or `flutter build ios`.
+You can also run in [Android Studio/IntelliJ][] or [VS Code][] with the Flutter and Dart plugins.
+This will run a single-view example version of your module without being
+embedded in your existing application. This can be useful for testing
+quick, incremental changes with hot reload.
 
-Assume you have an existing iOS app at `some/path/MyApp`, and that you
-want your Flutter project as a sibling:
+### Module organization
+The `my_flutter` module directory structure is similar to a normal Flutter
+application:
 
-```bash
-cd some/path/
-flutter create -t module my_flutter
+```text
+my_flutter/
+  .ios/
+    Runner.xcworkspace
+    Flutter/podhelper.rb
+  lib/
+    main.dart
+  test/
+  pubspec.yaml
 ```
 
-This creates a `some/path/my_flutter/` Flutter module project with some Dart
-code to get you started, and a `.ios/` hidden subfolder. The `.ios/` folder wraps
-the module project that contains some CocoaPods and a helper Ruby script.
+Add your Dart code to the `lib/` directory.
+
+You can add Flutter dependencies to `my_flutter/pubspec.yaml`, including Flutter plugins 
+with iOS platform code.
+
+There is a `.ios/` hidden subfolder which contains a Xcode workspace where you can 
+run a stand-alone version of your module. It also contains helper scripts to embed
+the module into your existing application with [CocoaPods][].
 
-## Make the host app depend on the Flutter module
+{{site.alert.note}}
+Custom iOS code should be added to your existing application or a plugin, not
+the module in `.ios/`. Changes made in `.ios/` will not be embedded in your existing application.
+The directory can be regenerated by running `flutter clean` in the `my_flutter` directory.
```

Comment:
Doesn't everything get overwritten too each time flutter packages get is run too? https://github.com/flutter/flutter/issues/28135

Reply:
That too https://github.com/flutter/flutter/issues/42956.